frontend/src/components/landing/DemoPhone.tsx

```diff
@@ -0,0 +1,45 @@
+import { useLocalization } from "@fluent/react";
+import styles from "./DemoPhone.module.scss";
+import BgImage from "../../../../static/images/newlanding/a/hero-image-bg.svg";
+import PremiumScreenshot from "../../../../static/images/newlanding/a/hero-image-premium.svg";
+import PremiumScreenshotFr from "../../../../static/images/newlanding/a/hero-image-premium-fr.svg";
+import PremiumScreenshotDe from "../../../../static/images/newlanding/a/hero-image-premium-de.svg";
+import NoPremiumScreenshot from "../../../../static/images/newlanding/a/hero-image-nopremium.svg";
+import FgImage from "../../../../static/images/newlanding/a/hero-image-fg.svg";
+import { getLocale } from "../../functions/getLocale";
+
+export type Props = {
+  premium?: boolean;
+};
+
+export const DemoPhone = (props: Props) => {
+  const { l10n } = useLocalization();
+  const lang = getLocale(l10n).split("-")[0] ?? "en";
+
+  // Show an English-language non-Premium interface by default:
+  let screenshot = NoPremiumScreenshot.src;
+  if (props.premium) {
+    // if Premium is available in the user's country,
+    // show the English-language Premium interface:
+    screenshot = PremiumScreenshot.src;
+    if (lang === "fr") {
+      // ...except if the user speaks French, in which case it will be localised in French:
+      screenshot = PremiumScreenshotFr.src;
+    }
+    if (lang === "de") {
+      // ...except if the user speaks German, in which case it will be localised in German:
+      screenshot = PremiumScreenshotDe.src;
+    }
```

Comment:
Hmm, we could, but it would introduce some non-standard syntax, since we'd have to look at the filesystem to see which images are available, and of course regular JS in the browser doesn't have a filesystem available. So I'd lean towards not doing this, especially given how much effort went into getting these in place (i.e. they're unlikely to need many changes).